recipes/sides/freaky-freekeh.md

```diff
@@ -0,0 +1,33 @@
+# Freaky Freekeh
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-20%25-green.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-0.3hrs-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Cheap-green.svg)
+![Serves Badge](https://img.shields.io/badge/Serves-8-green.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Green Badge Count Badge](https://img.shields.io/badge/Green%20badge%20count-100%25-green.svg)
+
++ **Chef:** Suha
++ **Sous:** Lubes
++ **Date:** Week 12 FACN2
+
+## Ingredients
+- 3 mugs of freekeh
+- oil
+- 1 large onion, grated
+- nutmeg
+- cardamom (preferably the spice, not the seeds)
+- salt
+- pepper
+- vegetarian chicken stock cube
+- flaked almonds, and/or other nuts of your choice
+
+## Method
+- cover the freekeh with water and leave for a couple of minutes until it absorbs the water
+- Heat some oil in a sauce pan, then add the grated onion, cardamom, nutmeg, salt, pepper and the stock, and leave for 2 minutes. **Suha's Special Tip:** You decide how much of each spice to use, but be generous with the cardamom ;) 
+- Add the freekeh, and cover with 3 mugs-worth of boiling water. Simmer gently, adding more water if necessary, tasting it regularly until it is ready. **Suha's Special Tip:** Be careful not to add too much water!
+- Fry or roast the flaked almonds, and serve them sprinkled on top of the freekeh.
+
+## Comments
```

Comment:
what is this ?

Reply:
lol

Reply:
in case people want to add comments, e.g. after they cook it or what they thought of it